app/lib/usecases/runs/GetAllRuns.js

```diff
@@ -21,12 +21,13 @@ const { runAdapter } = require('../../database/adapters');
 /**
  * Executes this use case.
  *
- * @param {Object} dto The GetRunsDto containing all data.
+ * @param {Object} query - object which defines filtering
  * @returns {Promise<Run[]>} Promise object represents the result of this use case.
  */
-const GetRunUsecase = async () => {
+// eslint-disable-next-line no-unused-vars
+const GetAllRunsUsecase = async (query) => { // TODO utitlization of that argument
```

Comment:
You should not add code that will serve at some point. If you do not use it now, do not add it now.

What I asked was simply to create a service class (such as `RunService` for example that provides a `getAll` method)